alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
```

Comment:
Eventually we'll get the new colors added to `vui-colors`, at which point I think we would just use them below, and include an `@apply(...);` similar to what was done in `vui-button`, since it provides a greater degree of flexibility than exposing variables for all the CSS properties used.